packages/venia-ui/lib/components/ProductOptions/option.js

```diff
@@ -1,62 +1,87 @@
-import React, { Component } from 'react';
+import React, { useCallback, useMemo } from 'react';
```

Comment:
Lots of changes in this file but mostly due to converting this to a functional component.